src/components/atoms/Cv/Cv.components.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { FC } from 'react';
+import { Text, BoxProps } from '../../quarks';
+
+export const CvHeading: FC<IHeadingProps> = ({ heading, sx, ...props }) => (
+  <Text
+    as="p"
+    sx={{ px: 2, mb: 3, mt: 4, borderRadius: 1, fontWeight: 'bold', bg: 5, ...sx }}
+    {...props}
+  >
+    {heading}
```

Comment:
po co tak? jak robisz {...props} na <Text> to childreny też zespreaduje, nie potrzebny dodatkowy props

Reply:
Podbijam, wtedy wystarczy również `FC<BoxProps>` zamiast nowego interface.